src/index.html

```diff
@@ -17,7 +17,7 @@
     integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0="
     crossorigin="anonymous" />
 </head>
-<body data-bind="event: { keypress: window.hotkeyHandler }">
+<body data-bind="event: { keyup: window.hotkeyHandler }">
```

Comment:
Reason?

Reply:
Related to the other hotkey event change. Keypress wasn't detecting the escape key.